src/common/mixins/scenario.js

```diff
@@ -75,16 +77,20 @@ var ScenarioMixin = {
 
   methods: {
 
-    validateYears()                   { return this.$sciris.validateYears(this) },
-    updateSets()                      { return this.$sciris.updateSets(this) },
-    exportGraphs()                    { return this.$sciris.exportGraphs(this) },
-    exportResults(datastoreID)        { return this.$sciris.exportResults(this, datastoreID) },
+    validateYears()                   { return utils.validateYears(this) },
+    updateSets()                      { return utils.updateSets(this) },
+    exportGraphs()                    { return utils.exportGraphs(this) },
+    exportResults(datastoreID)        { return utils.exportResults(this, datastoreID) },
     scaleFigs(frac)                   { return this.$sciris.scaleFigs(this, frac)},
     clearGraphs()                     { return this.$sciris.clearGraphs(this) },
     togglePlotControls()              { return this.$sciris.togglePlotControls(this) },
     getPlotOptions(project_id)        { return this.$sciris.getPlotOptions(this, project_id) },
```

Comment:
@parhammmm it looks like some of these are still atomica-specific...

Reply:
you mean these two?

```js
togglePlotControls()              { return this.$sciris.togglePlotControls(this) },
getPlotOptions(project_id)        { return this.$sciris.getPlotOptions(this, project_id) },
```